model/user.go

```diff
@@ -35,6 +37,7 @@ type User struct {
 	Username           string    `json:"username"`
 	Password           string    `json:"password"`
 	AuthData           string    `json:"auth_data"`
```

Comment:
Well, there is no API service that takes AuthData as a parameter and all it really is is the user's id for the oauth provider's system. The only reason to encrypt it would be to make sure no one can tell which mattermost account  is connected to which GitLab (or other provider) account